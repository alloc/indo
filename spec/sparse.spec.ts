test('sparse checkout', async () => {
  process.chdir('sparse')
  await indo()

  // Only "mdx/packages/mdx" is checked out.
  expect(fs.list('mdx')).toMatchObject(['.git', 'packages'])
  expect(fs.list('mdx/packages')).toMatchObject(['mdx'])

  // Are node_modules installed?
  expect(fs.isDir('mdx/packages/mdx/node_modules'))
})
