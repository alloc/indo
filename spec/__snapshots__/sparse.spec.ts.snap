// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sparse checkout 1`] = `
Array [
  "+ Cloned mdx from github.com/mdx-js/mdx",
  "✔︎ Installed node_modules of 1 package",
  "✔︎ Local packages are linked!",
]
`;
